src/Backend/Modules/Blog/Layout/Templates/Add.html.twig

```diff
@@ -155,8 +155,8 @@
           {{ macro.buttonIcon(geturl('Index'), 'times', 'lbl.Cancel'|trans|ucfirst, 'btn-default') }}
         </div>
         <div class="btn-group pull-right" role="group">
-          {{ macro.buttonIcon('', 'file-o', 'lbl.SaveDraft'|trans|ucfirst, 'btn-default', {"type":"submit", "id":"saveAsDraft"}) }}
-          {{ macro.buttonIcon('', 'plus-square', 'lbl.Add'|trans|ucfirst, 'btn-primary', {"id":"addButton", "type":"submit", "name":"add"}) }}
+          {{ macro.buttonIcon('', 'file-o', 'lbl.SaveDraft'|trans|ucfirst, 'btn-default', {"type":"submit", "name":"saveAsDraft"}) }}
+          {{ macro.buttonIcon('', 'plus-square', 'lbl.Publish'|trans|ucfirst, 'btn-primary', {"type":"submit", "name":"add"}) }}
```

Comment:
`name` should be `publish` and a translation instead of hardcoded text.

Reply:
I've altered the name to `publish` but using a translation would jeopardize the workings of the PHP bit since it can no longer rely on consistent data.

Reply:
ok